### ISSUE TITLE
[BUG] Module not found: Error: Can't resolve 'rc-picker/lib/generate/dayjs' 

### ISSUE THREAD

**Issue by vibl** [pankod/refine#2191]
**Describe the bug**
```
Compiled with problems:

ERROR in ./node_modules/@pankod/refine-antd/dist/esm/index.js 978:0-46

Module not found: Error: Can't resolve 'rc-picker/lib/generate/dayjs' in '/home/XXX/dev/js/tmp/refine-tutorial-ant/node_modules/@pankod/refine-antd/dist/esm'

``` 

**To Reproduce**
```
npx superplate-cli -p refine-react refine-tutorial-mui

# Default options except "Use Ant Design"

cd refine-tutorial-mui
npm run dev
``` 

**Desktop (please complete the following information):**
 - OS: Kubuntu 2022-04
 - Browser: Chrome 103

**Comment by vibl** [comment]
I've just realized this bug has already been filed: https://github.com/pankod/refine/issues/1793

But it was closed without being solved, so it's probably worth a new issue :-)

**Comment by vibl** [comment]
Solved after upgrading to Node v16.16.0 from v16.11.1.
